<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QDialog" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="ResultsTextEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ResultsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Resultat :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ParcourirButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Parcourir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compilationButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Compilation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="source">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
